### PR TITLE
Feature aogeoupdate

### PR DESCRIPTION
**Pull request by caitlinadams**
Finished first draft of updates to coastline identification notebooks.

### CODE REVIEW

File: Coastal_erosion_monitoring.ipynb [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Connect to the datacube\n",
+    "dc = datacube.Datacube(app='Coastal erosion')\n",
+    "\n",
+    "# Create the 'query' dictionary object, which contains the longitudes, latitudes and time provided above\n",
+    "query = {'y': lat_range,\n",
```
Comment (alexgleith) [inline Coastal_erosion_monitoring.ipynb]: Can we please indent these things in the PEP standard, so like:
```
dict = {
    'thing': 1,
    'thing2': 2
}
```

File: Coastal_erosion_monitoring.ipynb [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+    "         'x': lon_range,\n",
+    "         'time': time_range,\n",
+    "         'output_crs': 'EPSG:28352',\n",
+    "         'resolution': (-25, 25)}\n",
+    "\n",
+    "# Load Landsat 8 data for the time and area in the query. This may take several minutes, so please be patient.\n",
+    "landsat_ds = waterline_funcs.load_cloudmaskedlandsat(dc=dc, \n",
```
Comment (alexgleith) [inline Coastal_erosion_monitoring.ipynb]: Indentation here could go on the next line too.

File: Coastal_erosion_monitoring.ipynb [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+    "tide_range = (0.00, 3.50)\n",
+    "tide_file = 'tide_heights/dalyriver_-13.32_130.23_tides.csv'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load cloud-masked Landsat data\n",
+    "The first step in this analysis is to load in Landsat data for the `lat_range`, `lon_range` and `time_range` we provided above. The code below first connects to the datacube database, and then uses the `load_cloudmaskedlandsat` function to load in data from the Landsat 8 satellite for the area and time included in `lat_range`, `lon_range` and `time_range`. The function will also automatically mask out clouds from the dataset, allowing us to focus on pixels that contain useful data:"
```
Comment (alexgleith) [inline Coastal_erosion_monitoring.ipynb:134]: Perhaps we should add a note regarding how long this load will take. 30 seconds? 2 minutes?

File: Coastal_erosion_monitoring.ipynb [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+    "# Plot output shapefile\n",
+    "ax1 = contour_gdf.plot()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above plot is a basic visualisation of the contours returned by the `contour_extract` function. Given we now have the shapfile, we can use a more complex function to make an interactive plot for viewing the change in shoreline over time below."
```
Comment (alexgleith) [inline Coastal_erosion_monitoring.ipynb:359]: Replace shapfile with shapefile :-)

File: S1_Water_Classification.ipynb [hunk 5]
```diff
@@ -75,24 +86,21 @@
...
-    "dc = datacube.Datacube(app = app_name)"
+    "### Specify product information"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### List Available Products"
+    "Before loading the data, you'll need to specify which product you want to load. You should have found one Sentinel 1 product in the product list. In the next cell, replace `product_name` with the name you found in the available products list. You'll need to keep the quotation marks."
```
Comment (alexgleith) [inline S1_Water_Classification.ipynb]: Let's remove this, we don't need it.

Comment (alexgleith) [reply S1_Water_Classification.ipynb]: Actually, I see it is used... let's think about whether people need to know.

File: S1_Water_Classification.ipynb [hunk 6]
```diff
@@ -117,16 +142,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "product_preferences = dict(product = \"s1_gamma0_geotif_scene\",\n",
-    "                     output_crs = \"EPSG:4326\",\n",
-    "                     resolution = (0.00013557119,0.00013557119))"
+    "area_information = dict(latitude = latitude,\n",
```
Comment (alexgleith) [inline S1_Water_Classification.ipynb]: PEP8 no spaces around `=` and doesn't need a line break.

**Review by alexgleith — COMMENTED** [review]
Hey, keeping this here, but I'm not sure that it's that relevant. 

I've gone through both examples and I think they're really good, both of them.

File: Coastal_erosion_monitoring.ipynb (hunk 2 above)
Comment (caitlinadams) [reply Coastal_erosion_monitoring.ipynb]: The function name here is really long, so using the indenting you suggested above is really unreadable:
```
landsat_ds = waterline_funcs.load_cloudmaskedlandsat(
    dc=dc,
    query=query,
    platform='ls8',
    bands=['red', 'green', 'blue', 'swir1']
)
```

Comment (caitlinadams) [reply Coastal_erosion_monitoring.ipynb]: That being said, the longer I look at it, the more I get used to it. So happy to go with this if you think this is ok.

Comment (alexgleith) [reply Coastal_erosion_monitoring.ipynb]: Looks like the standard really [supports either way](https://github.com/python/typing/issues/433). Up to you here.

I like the one I suggested, and don't find it unreadable! It's probably personal taste, so whatever you go with is good, so long as we're consistent :-)

Comment (caitlinadams) [reply Coastal_erosion_monitoring.ipynb]: All good, decided to go with your suggestion. Lachlan also pointed out that it's less overhead to deal with the spacing if you change the function name, so definitely all for it in that respect!

File: S1_Water_Classification.ipynb (hunk 5 above)
Comment (caitlinadams) [reply S1_Water_Classification.ipynb]: I've decided to take this out. GA didn't seem that keen on it.

**Comment by caitlinadams** [comment]
Hi @alexgleith, would you mind having a final look? I've updated the Sentinel notebook better reflect PEP8.

**Pull request merged**